docs/user/json_data_converter_HOWTO.md

```diff
@@ -0,0 +1,536 @@
+
+Every object within an array will be flattened and treated separately. Nested arrays are then also flattened and treated separately. For example, using the JSON example above, we could have `resource_spans.[].scope_spans.[].spans.[].attributes.[].key` to access all the keys within the `attributes` array within the `spans` array within the `scope_spans` array within the `resource_spans` array which would provide the following exploded list of keys:
+
+```json
+["http.method", "http.host", "http.response"]
+```
+
+Many different fields will be accessed this way and as such the pathing structure is crucial to correctly access the data. Each field must be consistent with all other fields in the configuration file. Consistency primarily depends upon the nesting of arrays in the path. 
+
+With two field paths, if one accesses an object in a path three arrays deep and another accesses an object in a path in the same array then these will be consistent and each object will be matched at the same index in the array.
+For example, the following two paths are consistent with each other, and each object in the `attributes` will then match the `key` object and the `value.Value` object:
+ * `resource_spans.[].scope_spans.[].spans.[].attributes.[].key`
+ * `resource_spans.[].scope_spans.[].spans.[].attributes.[].value.Value`
+
+This produces the following exploded lists of keys for the first and second paths, respectively:
+```json
+["http.method", "http.response", "http.method", "http.response"]
+[{"StringValue": "GET"}, {"IntValue": "200"}, {"StringValue": "PUT"}, {"IntValue": "200"}]
+```
+
+If one field path accesses an object in a path four levels deep and another field path accesses an object in a path three levels deep then the second field will be duplicated for each object in the fourth level array.
+For example, the following two paths are consistent with each other and mean `.name` object will be duplicated across all the `.key` objects in the `attributes` array:
+ * `resource_spans.[].scope_spans.[].spans.[].attributes.[].key`
+ * `resource_spans.[].scope_spans.[].scope.name`
+
+This produces the following exploded lists of keys for the first and second paths, respectively:
+```json
+["http.method", "http.response", "http.method", "http.response"]
+["Group 1", "Group 1", "Group 1", "Group 1"]
+```
+### 3.2 field_mapping
+
+The `field_mapping` section is the core of the configuration, defining how to map fields from the input JSON to the application schema. Each field in the application schema is defined here. For example:
+
+```yaml
+field_mapping:
+    event_type:
+        key_paths: [
+            resource_spans.[].scope_spans.[].spans.[].attributes.[].key
+        ]
+        key_value: [http.method]
+        value_paths: [value.Value.StringValue]
+        value_type: string
+```
+
+This example would return `GET` from the [JSON example](#json-example).
```

Comment:
Specify that this is what it would return for trace001, as trace002 will return PUT